pennylane/hf/hamiltonian.py

```diff
@@ -352,6 +353,54 @@ def _generate_qubit_operator(op):
     return c, o
 
 
+def _simplify(h, cutoff=1.0e-12):
```

Comment:
Why not make this visible to users? (So `simplify()`, not `_simplify()`)
Seems like something that could be useful more generally in pennylane.

@josh146 Mentioned today that such general functionality merits thought on where to best implement it. Does it make sense for simplify to exist outside of the `hf` module? Are we simply waiting on the operator refactor to do so?

Reply:
Please see my comment below.